compose/compose-file/compose-file-v2.md

```diff
@@ -1415,6 +1415,49 @@ refer to it within the Compose file:
         external:
           name: actual-name-of-network
 
+#### host or none
+
+Use the host's networking stack, or no networking. Equivalent to
+`docker run --net=host` or `docker run --net=none`. Only used if you use
+`docker stack` commands. If you use the `docker compose` command,
```

Comment:
`docker-compose` (not `docker compose`)

Since this is the reference for v2, and v2 is not supported by `docker stack` commands, do we need this here? It seems like it's only appropriate for the v3 reference.